marlowe-cli/src/Language/Marlowe/CLI/Test/Script.hs

```diff
@@ -348,15 +351,16 @@ interpret so@Publish {..} = do
       pure marloweScriptRefs
     Nothing -> do
       logSoMsg' so "Scripts not found so publishing them."
-      Seconds timeout <- view seTransactionTimeout
+      -- Seconds timeout <- view seTransactionTimeout
+      executionTimeout <- view seExecutionTimeout
       runSoCli so $ publishImpl
         connection
         waSigningKey
         Nothing
         waAddress
         publishingStrategy
         (CoinSelectionStrategy False False [])
-        timeout
```

Comment:
@paluh I noticed here we weren't using `(Just timeout)` here. Maybe we need to update `publishImpl` to accept a Maybe for this argument, otherwise the simulation feature will not work correctly for this function, right?